### ISSUE TITLE
[BUG] ENHANCE THE USER INERFACE OF THE LOGIN FORM AND MADE IT INTERACTIVE

### ISSUE THREAD

**Issue by dnyanesh99borse** [swaraj-das/Collect-your-GamingTools#490]
### Description

**### ENHANCE THE USER INERFACE OF THE LOGIN FORM AND MADE IT INTERACTIVE**

The login form will enhance by incorporating advanced CSS effects, making it visually appealing and more interactive. Key improvements include hover and focus effects on input fields and buttons, creating a more engaging user experience. The input fields now change border color and add a subtle box-shadow when focused, while icons inside the fields change color for better visual feedback. The form background is transparent, allowing a custom background image on the body for a modern look. Buttons have smooth transitions and scaling effects, and full form validation ensures proper user input, making the login process seamless and user-friendly.

### Screenshots

![image](https://github.com/user-attachments/assets/9a4e2e7a-c5ff-4eaa-8554-2c65a4a99b1e)

### Any additional information?

**SIR I HAD ACTUALLY RESOLVE THIS ISSUE AND ALREADY PULL THE REQUESTE AND JUST THE ONE MINUTE BEFORE THAT YOU CLOSED THE ISSUE. AND THAT ISSUE WAS ALSO CREATED BY ME. SIR I BELEIVE MY UPDATIONS WILL GIVE THE NEXT LEVEL EXPERIENCE I AM REQUESTING YOU TO PLEASE ONE REVIEW THE CHANGES AND RESPECT MY HARDWORK @swaraj-das #419** #488 

### Checklist

- [X] I have checked the existing [issues](https://github.com/swaraj-das/Collect-your-GamingTools/issues).
- [X] I have read the [Code Of Conduct](https://github.com/swaraj-das/Collect-your-GamingToolst/blob/main/CODE_OF_CONDUCT.md)
- [X] The changes don't break the code
- [X] Please attach the screenshots and video of the development work once it is done

**Comment by dnyanesh99borse** [comment]
SIR PLEASE ASSIGN THIS ISSUE TO ME AND JUST ONCE REVIEW THE PR I HAD MADE IT WILL DEFINATELY LIKE YOU . I BELEIVE. PLEASE SIR PLEASE. PLEASE @swaraj-das @swaraj-das

**Comment by swaraj-das** [comment]
@dnyanesh99borse There is already a login page which is good, You can enhance the sign up page.

**Comment by biswa2005** [comment]
i have already done it #419

**Comment by dnyanesh99borse** [comment]
SIR ONCE YOU PLEASE CHECK THE CHANGES I HOPE YOU WILL DEFINATELY LIKE IT AND PLEASE RESPECT MY EFFORTS PLEASE SIR PLEASE. @swaraj-das
